ci/lint.sh

```diff
@@ -49,6 +49,11 @@ if [ "$LINT" ]; then
     if [ $? -ne "0" ]; then
         RET=1
     fi
+
+    if [[ -n $(find **/*.pyx -type f -exec egrep -l " +$" {} \;) ]]
```

Comment:
Will this print the list of offending files? Can you also include .pxd, .pxi, and .pxi.in?  Thanks for stepping up on this, linting these files is really tough.

Reply:
Yea no worries! It won't display them as output I think just from this test but I can include the same find / grep dance under it or save it as a variable and echo it.

Reply:
> but I can include the same find / grep dance under it or save it as a variable and echo it.

Great.  If something breaks, we definitely want to know where to look for it.